axelor-account/src/main/java/com/axelor/apps/account/service/moveline/massentry/MoveLineMassEntryRecordServiceImpl.java

```diff
@@ -258,4 +264,45 @@ public MoveLineMassEntry setInputAction(MoveLineMassEntry moveLine, Move move) {
     }
     return moveLine;
   }
+
+  @Override
+  public void setAnalyticMoveLineMassEntryList(
```

Comment:
Can you rename this to `fillAnalyticMoveLineMassEntryList`

Reply:
Resolved